fast64_internal/render_settings.py

```diff
@@ -191,30 +184,62 @@ class Fast64RenderSettings_Properties(bpy.types.PropertyGroup):
         default=(0.5, 0.5, 0.5, 1),
         update=on_update_render_preview_nodes,
     )
-    lightColor: bpy.props.FloatVectorProperty(
-        name="Light Color",
+    light0Color: bpy.props.FloatVectorProperty(
+        name="Light 0 Color",
         subtype="COLOR",
         size=4,
         min=0,
         max=1,
         default=(1, 1, 1, 1),
         update=on_update_render_preview_nodes,
     )
-    lightDirection: bpy.props.FloatVectorProperty(
-        name="Light Direction",
+    light0Direction: bpy.props.FloatVectorProperty(
+        name="Light 0 Direction",
         subtype="DIRECTION",
         size=3,
         min=-1,
         max=1,
-        default=mathutils.Vector((0.5, 0.5, 1)).normalized(),  # pre normalized
+        default=mathutils.Vector((1.0, -1.0, 1.0)).normalized(),  # pre normalized
+        update=on_update_render_preview_nodes,
+    )
+    light0SpecSize: bpy.props.IntProperty(
+        name="Light 0 Specular Size",
+        min=1,
+        max=255,
+        default=3,
+        update=on_update_render_preview_nodes,
+    )
+    light1Color: bpy.props.FloatVectorProperty(
+        name="Light 1 Color",
+        subtype="COLOR",
+        size=4,
+        min=0,
+        max=1,
+        default=(0, 0, 1, 1),
```

Comment:
shouldn´t the default be black? i noticed this before but just thought it didn´t matter but then I remembered that sm64 lighting engine exists aha, by default it has only one global light direction

Reply:
Fixed